lambda/aws-contentgen-appsync-lambda/src/image_generator.py

```diff
@@ -0,0 +1,283 @@
+import boto3,os,base64,json
+import requests as reqs
+from aws_lambda_powertools import Logger, Tracer, Metrics
+from requests_aws4auth import AWS4Auth
+from datetime import datetime
+
+logger = Logger(service="IMAGE_GENERATION")
```

Comment:
```suggestion
logger = Logger(service="CONTENT_GENERATION")
tracer = Tracer(service="CONTENT_GENERATION")
metrics = Metrics(namespace="content_generation", service="CONTENT_GENERATION")
```

Reply:
done in 3f90ec921befd1a503e473dd0a582f072c69f76b